File: app-presentation/src/main/java/com/zxjdev/smile/presentation/application/util/ui/ErrorMessagePrinter.java
package com.zxjdev.smile.presentation.application.util.ui;

import android.content.Context;
import android.widget.Toast;

/**
 * Shows error messages to user
 */
public class ErrorMessagePrinter {

  private Context context;

  public ErrorMessagePrinter(Context context) {
    this.context = context;
  }

  public void print(String message) {
    Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
  }
}

File: app-presentation/src/main/java/com/zxjdev/smile/presentation/common/util/ui/ErrorMessagePrinter.kt
package com.zxjdev.smile.presentation.common.util.ui

import android.content.Context
import android.widget.Toast

/**
 * Shows error messages to user
 */
class ErrorMessagePrinter(private val context: Context) {

    fun print(message: String?) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
    }
}
